config/runtimes/kserve-sklearnserver.yaml

```diff
@@ -10,13 +10,6 @@ spec:
     - name: kserve-container
       image: kserve-sklearnserver:replace
       args:
-        - --model_name=model
+        - --model_name={{.Name}}
         - --model_dir=/mnt/models
         - --http_port=8080
```

Comment:
okay, @yuzisun. But we will keep the current hardcoded resource defaults logic in mutating webhook for now since it may be used in multi-model deployment.